lib/intern.js

```diff
@@ -0,0 +1,160 @@
+const _ = require('lodash')
+const Uuid = require('node-uuid')
```

Comment:
we really should provide this for stores - I have added an issue to seneca-entity to look at this in the next release
not urgent